<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialMediaApp</class>
 <widget class="QMainWindow" name="SocialMediaApp">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>46</red>
          <green>46</green>
          <blue>46</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>46</red>
          <green>46</green>
          <blue>46</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>46</red>
          <green>46</green>
          <blue>46</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>46</red>
          <green>46</green>
          <blue>46</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>46</red>
          <green>46</green>
          <blue>46</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>46</red>
          <green>46</green>
          <blue>46</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>46</red>
          <green>46</green>
          <blue>46</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>46</red>
          <green>46</green>
          <blue>46</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>46</red>
          <green>46</green>
          <blue>46</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(46, 46, 46, 255));
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>1170</width>
      <height>650</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget 
{
background: transparent; 
}
QStackedWidget:item
{
color:white;
}</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="Home">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTextBrowser" name="HomeText">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>9</y>
        <width>1150</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Timeline">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTextBrowser" name="TimelineText">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>9</y>
        <width>1150</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="FriendList">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTextBrowser" name="FriendListText">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>9</y>
        <width>1150</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFunctions">
    <property name="title">
     <string>Functions</string>
    </property>
    <addaction name="actionSet_User"/>
    <addaction name="actionLike_Post"/>
    <addaction name="actionComment"/>
    <addaction name="actionShare_Memory"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionView_Post"/>
    <addaction name="actionView_Timeline"/>
    <addaction name="actionView_Friend_List"/>
    <addaction name="actionView_Page"/>
   </widget>
   <addaction name="menuFunctions"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionSet_User">
   <property name="text">
    <string>Set User</string>
   </property>
  </action>
  <action name="actionLike_Post">
   <property name="text">
    <string>Like Post</string>
   </property>
  </action>
  <action name="actionComment">
   <property name="text">
    <string>Comment</string>
   </property>
  </action>
  <action name="actionShare_Memory">
   <property name="text">
    <string>Share Memory</string>
   </property>
  </action>
  <action name="actionView_Post">
   <property name="text">
    <string>Post</string>
   </property>
  </action>
  <action name="actionView_Timeline">
   <property name="text">
    <string>Timeline</string>
   </property>
  </action>
  <action name="actionView_Friend_List">
   <property name="text">
    <string>Friend List</string>
   </property>
  </action>
  <action name="actionView_Page">
   <property name="text">
    <string>Page</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
